src/krkn_lib/tests/test_utils.py

```diff
@@ -323,3 +324,66 @@ def test_is_host_reachable(self):
         self.assertFalse(
             is_host_reachable(f"{get_random_string(10)}.com", 12345)
         )
+
+    def test_get_junit_test_case(self):
+        test_suite_name = "krkn-test"
+        test_case_description_success = (
+            "success [test-case] krkn-lib test case"
+        )
+        test_case_description_failure = (
+            "failure [test-case] krkn-lib test case"
+        )
+        test_stdout = "KRKN STDOUT"
+        test_version = "OCP 4.16"
```

Comment:
Have a [pr](https://github.com/krkn-chaos/krkn-lib/pull/115) to add the ocp version to the telemetry, would be good if we could get the ocp version from there

Reply:
sorry, didn't realize this was in a test. Ignore this comment